### PR TITLE
Feat/reverse navigation

### LINKED ISSUE #2091: Navigation -Create ascending Navigation page
Provide content for reverse navigation to be placed on the main Navigation page

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-2187--keen-mayer-a86c8b.netlify.app/templates/reverse-navigation)

#### What does this PR do?
Create the reverse Navigation page
#### Where should the reviewer start?
reverse navigation
#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### Any background context you want to provide?
Since the diagrams used in the figma file need to use the arrows and currently we do not have arrows as a option in our grommet Diagram component I went ahead and used the images. I created a new component that will take the images and then determine the light/dark mode. This is all wrapped in our `figurewrapper`
#### What are the relevant issues?
Removed reverse nav 
and sets up Image component to be used
#### Should this PR be mentioned in Design System updates?
yes
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: aries-site/src/pages/templates/reverse-navigation.mdx [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+## When to Use:
+
+- When the user may want to return to the previous page’s content
+- When the user may want to trace through their steps chronologically to return to past page content
+- For web applications, as all web browsers contain built in chronological content buttons
+
+## Reverse Navigation Examples:
+
+### In-App Example
+Provide a context button for when it is valuable to provide users explicitly around where they are returning to.
```
Comment (taysea) [inline aries-site/src/pages/templates/reverse-navigation.mdx:16]: I had left a few comments on the Figma file re: wording suggestions on this page -- we can wait for Vicky's replies there.

Comment (britt6612) [reply aries-site/src/pages/templates/reverse-navigation.mdx:16]: sounds good I had left what I had until I had a chance to go over with Vicky

File: aries-site/src/pages/templates/reverse-navigation.mdx [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+
+<Example caption="Diagram illustrating reverse navigation." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
```
Comment (taysea) [inline aries-site/src/pages/templates/reverse-navigation.mdx:18]:
```suggestion
<Example caption="Diagram showing an in-app back button that provides the user with context around where they will return to after clicking the button." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
```

File: aries-site/src/pages/templates/reverse-navigation.mdx [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+<DiagramPreview src={{dark: "/BackButton.svg", light: '/BackButton-invert.svg'}} alt="Diagram illustrating the reverse navigation." />
```
Comment (taysea) [inline aries-site/src/pages/templates/reverse-navigation.mdx:19]:
```suggestion
<DiagramPreview src={{dark: "/BackButton.svg", light: '/BackButton-invert.svg'}} alt="Diagram showing an in-app back button that provides the user with context around where they will return to after clicking the button." />
```

**Review by taysea — COMMENTED** [review]
The png files look a little blurry -- I'm not sure if it has to do with the size they were exported as. Maybe we try exporting them as 2x?

**Comment by britt6612** [comment]
> The png files look a little blurry -- I'm not sure if it has to do with the size they were exported as. Maybe we try exporting them as 2x?

I made them on my own so I can try that if that doesn't look better I can ask Vicky.

File: aries-site/src/pages/templates/reverse-navigation.mdx [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+import { DiagramPreview, Example } from '../../layouts';
+import {
+  NameValueListFontWeightSecondPreview,
+} from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+
+- When the user may want to return to the previous page’s content
```
Comment (ericsoderberghp) [inline aries-site/src/pages/templates/reverse-navigation.mdx:9]: I don't think this is an accurate description of the use of this pattern. I thought we were leaning more towards the term "Context" or "Up" to indicate that the user would be taken to the page containing this page in the overall site hierarchy. If the user did not navigate to this page from that containing page, then clicking the button would not return to the "previous" page, nor is it aligned to anything "chronological".

File: aries-site/src/pages/templates/reverse-navigation.mdx [hunk 5]
```diff
@@ -1,3 +1,36 @@
-import { ComingSoon } from '../../components';
+import { Image } from 'grommet';
+import { DiagramPreview, Example } from '../../layouts';
+import {
+  NameValueListFontWeightSecondPreview,
+} from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
```
Comment (halocline) [inline aries-site/src/pages/templates/reverse-navigation.mdx:7]: For consistency, we haven't been using ":" in headings.

```suggestion
## When to Use
```

or we could do something like:
```suggestion
## When to Use

Reverse navigation should be applied when:
```

File: aries-site/src/pages/templates/reverse-navigation.mdx (hunk 4 above)
Comment (halocline) [reply aries-site/src/pages/templates/reverse-navigation.mdx:9]: @vavalos5 @britt6612 @ericsoderberghp 

I am a bit confused on which pattern we are presenting here.

Are we presenting:
1) a pattern which allows a user to step back through their history?
OR
2) a pattern which allows a user to navigate from child to parent / from spoke to its hub?


Clarification on this point will help me review.

File: aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js [hunk 6]
```diff
@@ -9,10 +9,16 @@ import { Box, Text } from 'grommet';
 export const NameValueListFormLabel = ({ data, name }) => {
   const theme = useContext(ThemeContext);
   return (
-    <Box margin={{ top: 'small' }} >
+    <Box margin={{ top: 'small' }}>
```
Comment (halocline) [inline aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js:12]: Is this file being touched just because of prettier formatting? Seems unrelated to this work.

**Review by halocline — CHANGES_REQUESTED** [review]
I'm going to hold off on reviewing further until we get confirmation on the purpose of this page and pattern we are presenting.

File: aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js (hunk 6 above)
Comment (britt6612) [reply aries-site/src/examples/components/namevaluelist/Examples/components/NameValueListFormLabel.js:12]: yes it was flagged on my eslint when I ran the `commit`

**Comment by britt6612** [comment]
Reverse Navigation is going to be removed so will no longer need its own card/page

**Review by halocline — APPROVED** [review]
No concerns. Looks good to go.

**Comment by vavalos5** [comment]
deploy preview at the top doesn't work, should i just go off of the netlify link?

**Comment by britt6612** [comment]
> Details

https://github.com/grommet/hpe-design-system/pull/2187#issuecomment-1010252646
try this link this is just removing the matrix nav

File: aries-site/src/pages/templates/navigation.mdx [hunk 7]
```diff
@@ -6,7 +6,6 @@ import { getPageDetails, getRelatedContent } from '../../utils';
 export const navPatterns = [
   'Drill Down Navigation',
   'Side to Side Navigation',
```
Comment (taysea) [inline aries-site/src/pages/templates/navigation.mdx:8]: I can't comment down below but in the copy we say "There are 4 different types of navigational directions..." -- ssince we removed "Reverse", we need to change this to "There are 3 different types..."

Comment (britt6612) [reply aries-site/src/pages/templates/navigation.mdx:8]: thanks done

**Review by taysea — APPROVED** [review]
